elblog.go

```diff
@@ -176,6 +205,9 @@ func scan(data []byte, atEOF bool) (advance int, token []byte, err error) {
 	}
 	// If we're at EOF, we have a final, non-empty, non-terminated word. Return it.
 	if atEOF && len(data) > start {
+		if trim && !open {
+			return len(data), data[start+1 : len(data)-1], nil
+		}
```

Comment:
This fixes a minor issue where we wouldn't trimming the last token if it needed to be trimmed.